pkg/app/api/service/pipedservice/service.proto

```diff
@@ -345,3 +352,21 @@ message ReportApplicationLiveStateEventsRequest {
 message ReportApplicationLiveStateEventsResponse {
     repeated string failed_ids = 1;
 }
+
+message GetLatestEventRequest {
+    string name = 1 [(validate.rules).string.min_len = 1];
+    map<string,string> labels = 2;
+}
+
+message GetLatestEventResponse {
+    pipe.model.Event event = 1 [(validate.rules).message.required = true];
+}
+
+message ListEventsRequest {
+    // returns all events created after this value.
+    int64 after = 1 [(validate.rules).int64.gt = 0];
```

Comment:
How about making this to be more familiar, by using `from` and `to`.
The response will contain all events in this range `[from, to)`. 
(from <= && < to) 
(And the mark at the piped should be `to`)

Reply:
Looks good. Then PIped uses to+1 value for the next right?

Reply:
That can be confusable, I mean for the next time Piped uses the maximum created_at in the previous result.

Reply:
I think piped uses `to`, because that the previous call, `to` was not included.

Reply:
And by this way we don't need to find the maximum created_at value from the previous response, Just use time.Now()

Reply:
I see, maybe I don't fully understand the standard about queries.

Reply:
Ok, let me confirm this is correct or not: `from` <= response < `to`

Reply:
- first call: [time.Now - time.Hour, time.Now]
- from the second call:  [previousCall.To, time.Now]

Reply:
>Ok, let me confirm this is correct or not: from <= response < to

Yes, that is correct.

Reply:
Fine. Definitely that is more familiar for most developers.